setup.py

```diff
@@ -413,7 +413,7 @@ def write_version(filename: str = str(AIRFLOW_SOURCES_ROOT / "airflow" / "git_ve
     "blinker",
     "bowler",
     "click>=8.0",
-    "coverage",
+    "coverage>=7.2",
```

Comment:
Just in case, [exclude_also](https://coverage.readthedocs.io/en/latest/config.html#report-exclude-also) introduced in 7.2.0